babel/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "example-babel-app",
+  "version": "1.0.0",
+  "description": "Babel example",
+  "main": "lib/index.js",
+  "scripts": {
+    "compile": "babel src -d lib",
+    "test": "mocha"
+  },
+  "directories": {
+    "lib": "./src",
+    "test": "./test"
+  },
+  "dependencies": {
```

Comment:
Should all of these packages be `devDependencies`?
After transpilation, are any of them needed to load/use the package?